src/javascript/app/pages/user/account/settings/professional_client.js

```diff
@@ -33,6 +33,10 @@ const professionalClient = (() => {
             return;
         }
 
+        $('#professional').setVisibility(0);
+        $('#processing').setVisibility(0);
+        $('#rejected').setVisibility(0);
```

Comment:
we are fetching the selectors here and also in the setvisible function, it would be better to store them once and reuse them